app.js

```diff
@@ -34,5 +35,10 @@ app.use(restaurantRoutes);
 app.use(menuRoutes);
 app.use(orderRoutes);
 app.use(paymentRouter);
-
+ // This middleware should be added before calling `applyMiddleware`.
+app.use(graphqlUploadExpress());
+app.use(express.static("public"))
+// app.listen({port:80}, ()=>{
```

Comment:
which port is the application going to listen to now?

Reply:
I remove app.listen()